internal/packages/changelog/yaml.go

```diff
@@ -100,19 +103,19 @@ func SetManifestVersion(d []byte, version string) ([]byte, error) {
 	var node yaml.Node
 	err := yaml.Unmarshal(d, &node)
 	if err != nil {
-		return nil, errors.Wrap(err, "failed to decode manifest")
+		return nil, fmt.Errorf("failed to decode manifest: %w", err)
 	}
 
 	// Manifest is a document, with a single element, that should be a map.
 	if len(node.Content) == 0 || node.Content[0].Kind != yaml.MappingNode {
-		return nil, errors.Wrap(err, "unexpected manifest content")
+		return nil, fmt.Errorf("unexpected manifest content: %w", err)
 	}
```

Comment:
This is not right; err is nill here but this returns a non-nil error referring to err.

Reply:
Thanks, good catch.